integrations/airflow/marquez_airflow/utils.py

```diff
@@ -157,6 +158,10 @@ def get_custom_facets(task, is_external_trigger: bool):
     }
 
 
+def new_run_id(dag_run_id: str, task_id: str) -> str:
```

Comment:
Does this function needs parameters now?

Reply:
I added the parameters so that the tests could override the function. I suppose I could have changed the test to return a list of expected return values, but I thought it would be simpler with a deterministic function so we don't have to keep track of which ids are returned in which order.